recipes/hssm/meta.yaml

```diff
@@ -0,0 +1,109 @@
+{% set version = "0.2.0" %}
+{% set wfpt_version = "0.1.4" %}
+
+package:
+  name: hssm
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/h/hssm/hssm-{{ version }}.tar.gz
+    sha256: a4b874ff970cdef38ca1d6ec738edb1708b6a9b5e5243b7e4217d4a428be4842
+    folder: hssm
+  - url: https://pypi.io/packages/source/h/hddm-wfpt/hddm_wfpt-{{ wfpt_version }}.tar.gz
+    sha256: b16d318ce47f4691b90e341f29918fd0f7ca0cbce120f794d2e7c8198a40fb6d
+    folder: hddm_wfpt
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+
+outputs:
+  - name: hddm-wfpt
+    version: {{ wfpt_version }}
+    build:
+      skip: true  # [py<=39 or py>=312 or py2k]
+      script: python -m pip install ./hddm_wfpt -vv --no-deps --no-build-isolation
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+      host:
+        - python
+        - setuptools
+        - wheel
+        - cython
+        - numpy >=1.23.4,<1.26.4
+        - pip
+      run:
+        - python
```

Comment:
add:
```
run:
  - {{ pin_subpackage("hssm-core", exact=True) }}
```

Reply:
Tried this, but when the tests for `hssm-core` ran, it could not find `hddm-wfpt`.

Since `hssm-core` depends on `hddm-wfpt`, I pinned `hddm-wfpt` in `hssm-core` instead. Tests now fail because `hssm-core` requires a py38 build of `hddm-wfpt`, which is confusing because python 3.8 appears in none of the specifications. Any tips on resolving this?

Thank you @bollwyvl!